koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -176,11 +175,12 @@ def clean(self):
 
 
 class LabelsWrapper(object):
-    def __init__(self, dockerfile_path, logger_name=None):
+    def __init__(self, dockerfile_path, logger_name=None, labels_override=None):
         self.dockerfile_path = dockerfile_path
         self._setup_logger(logger_name)
         self._parser = None
         self._label_data = {}
+        self._labels_override = labels_override
```

Comment:
Actually I think default to None is better than default to {}. Better to have immutable types for method argument defaults.